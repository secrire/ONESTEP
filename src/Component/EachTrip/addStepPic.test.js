import React from "react"; 
import AddStepPic from "./addStepPic"; 
import renderer from 'react-test-renderer'; 

it("react render", () => { 
    const component = renderer.create( 
        <AddStepPic/>); 
        let tree = component.toJSON(); 
        expect(tree).toMatchSnapshot(); 
    });
