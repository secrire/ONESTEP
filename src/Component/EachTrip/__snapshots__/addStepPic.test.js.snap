// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react render 1`] = `
<label
  className="step-pic-label"
>
  <input
    className="trip-cover-change-pic"
    id="uploadPicInput"
    onChange={[Function]}
    type="file"
  />
  <img
    className="step-upload-pic-icon"
    src="./imgs/bluecamera.svg"
  />
</label>
`;
